client/src/components/Game/Canvas.js

```diff
@@ -256,7 +257,8 @@ class Canvas extends Component {
             {(index !== usernames.length - 1) ? <span>&rarr;</span> : null}</span>
           ))}
         </div>
-
+        {timeLimit && userIndex === currentPlayerIndex && <Timer />}
+        <h4>Draw something based on the left image!</h4>
         <p className="title">Draw something based on the left image!</p>
```

Comment:
Looks like some duplication due to merging from master not auto-resolving correctly. have you tested how it looks on the dev server after merging from master?